Realm/RLMSchema.h

```diff
@@ -30,7 +30,7 @@ RLM_HEADER_AUDIT_BEGIN(nullability, sendability)
 
  Schemas map to collections of tables in the core database.
  */
-RLM_SWIFT_SENDABLE
+RLM_SWIFT_SENDABLE // not actually immutable, but the public API kinda is
```

Comment:
Is it dangerous in any way having the schema classes partially mutable given the above comment?

Reply:
Yes, but it's not a new problem. This is a place where Sendable annotations are pointing out that we're doing something which requires care. All of the mutation needs to happen before we expose the objects as part of the shared schema, and there's nothing which will warn us if we get that wrong.